src/main/webapp/creatives/maps/simid_map_creative.test.js

```diff
@@ -0,0 +1,64 @@
+/** Tests to see if Find Nearest X button is appending correct text */
+
+import SimidMapCreative from './simid_map_creative.js';
+
+test('testing button given ad params', ()=> {
```

Comment:
Make sure these names answer the question "What is being tested"

In this case, you're testing that the button text gets updated according to the ad params, so maybe a name like `testing button text updates from ad parameters`

Reply:
Okay, I will make it a little clearer